app/models/comment.rb

```diff
@@ -0,0 +1,14 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  after_create :increment_commentscounter
```

Comment:
- [ ] Instead of using the `after_create` callback method, let us use the more appropriate `after_save` callback method as we will be using these methods upon saving posts or comments and likes and not upon creating them.

## This applies to the Like and Post model as well